webpack.config.js

```diff
@@ -0,0 +1,75 @@
+const path = require('path')
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+
+module.exports = {
+    entry: './src/',
+    context: __dirname,
+
+    output: {
+        path: path.resolve(__dirname, 'build'),
+        filename: 'bundle.js'
+    },
+    module: {
+        rules: [
+
+            /*
+            {
+              test: /\.(js|jsx)$/,
+              exclude: /node_modules/,
+              loader: 'eslint-loader?fix=true',
+              enforce: 'pre'
+            },
+            */
+
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                loader: 'babel-loader'
+            },
+
+            {
+                test: /\.css$/,
+                loader: ['style-loader', 'css-loader']
+            },
+
+            {
+                test: /\.scss$/,
+                loaders: ['style-loader',
+                    'css-loader?sourceMap&-minimize&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]',
```

Comment:
Almost all of it. Sourcemaps are for mapping the css back to the original sass content when viewing it in developer tools in the browser. The minimize param minimizes the css. importLoaders i am a bit uncertain about, but bascially the loader itself (css-loader that is) takes care of a css file and this param tells css-loader how many other loaders should be applied when `@import` statements occurs in the CSS files itself. https://webpack.js.org/loaders/css-loader/#importloaders

`localIdentName` is used to resolve how clasname should be generated when using classname references inside a component that imports a sass or css file:
```javascript
import styles from './styles.css';
...
render() {
  return <p classNam={styles.fooBar}>FooBar</p>
}
```

That will bascially result in `<p class="fooBar_gh292u2_hashelihash">FooBar</p>`. Its a sort of local namespace thingie for imported css classes.

I saw that there have been some changes to WebPack and how you provide params to the loaders since i did this so im gonna take a look and update my knowledge about the API.